### DISCUSSION TITLE
Data related to how many people are using GitHub Actions

### DISCUSSION THREAD

**Discussion by beatrizmilz** [community/community#26315] [Actions]
<p>Hi!<br>
I’m creating a lightning talk (5 minutes) for a conference and I’m seaching for some data (or visualization made by GitHub) about the use of GitHub Actions.<br>
I want to add in my presentation a visualization regarding how the use of GH Actions is increasing in the recent years.<br>
I searched for it, but found nothing. If anyone knows where I can find data or graphs related to this, would be great :slight_smile: Thanks!</p>

**Comment by mpboom** [comment, marked as answer]
<p>Sure!</p>
<ul>
<li><a href="https://octoverse.github.com/writing-code-faster/#scale-through-automation" class="inline-onebox" rel="noopener nofollow ugc">Writing and shipping code faster | The State of the Octoverse</a></li>
<li><a href="https://octoverse.github.com/static/octoverse-report-2021.pdf" rel="noopener nofollow ugc">https://octoverse.github.com/static/octoverse-report-2021.pdf</a></li>
</ul>

**Comment by beatrizmilz** [comment]
<p>Thank you SO MUCH! :smiley:</p>
